Makefile

```diff
@@ -98,7 +102,7 @@ configs-integration-test: build-image/$(UPTODATE)
 		-v $(shell pwd)/.cache:/go/cache \
 		-v $(shell pwd)/.pkg:/go/pkg \
 		-v $(shell pwd):/go/src/github.com/cortexproject/cortex \
-		-v $(shell pwd)/cmd/configs/migrations:/migrations \
+		-v $(shell pwd)/migrations:/migrations \
```

Comment:
@tomwilkie this change is actually only in the in the integration test container. The build container does not mount the migrations

Reply:
+1